src/main/java/lotto/domain/port/in/LottoUseCase.java

```diff
@@ -0,0 +1,12 @@
+package lotto.domain.port.in;
+
+import lotto.application.dto.LottoResultDTO;
+import lotto.domain.model.LottoTickets;
+import lotto.domain.model.WinningLotto;
+
+public interface LottoUseCase {
+    void purchase(int amount);
+    LottoTickets getLottoTickets();
+
+    LottoResultDTO checkWinning(LottoTickets tickets, WinningLotto winningLotto, int amount);
```

Comment:
```suggestion
    void purchase(int amount);
    LottoTickets getLottoTickets();
    LottoResultDTO checkWinning(LottoTickets tickets, WinningLotto winningLotto, int amount);
```
사소한 줄바꿈도 지켜주면 좋을 것 같아요